### DISCUSSION TITLE
Curso Dominando Injeção de Dependência - Formas de registrar uma dependencia

### DISCUSSION THREAD

**Discussion by JoabeRochadel** [balta-io/forum#687] [Tira Dúvidas] [answered]
Boa tarde Equipe Balta, parabéns pelo conteúdo sempre muito esclarecedor.

Gostaria de pontuar que em alguns projetos eu já usei um padrão que não sei se era a melhor forma, mas eu consegui resolver que era usando o ServiceLocator. Pelo que entendi, ele é próximo do que é usado na aula **[Resolvendo dependências via HttpContext]**

**Comment by andrebaltieri** [comment, marked as answer]
Bom dia, @JoabeRochadel, como vai?

Não sei se entendi muito bem sua dúvida... ficou algo na aula para implementar?
Se puder esclarecer melhor o cenário, posso tentar ajudar mais.

Abraços e obrigado pelos feedbacks 🚀
